ext/phar/tests/tar/bug71504.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Bug #71504: Parsing of tar file with duplicate filenames causes memory leak
+--SKIPIF--
+<?php if (!extension_loaded('phar')) die('skip'); ?>
+<?php if (!extension_loaded("spl")) die("skip SPL not available"); ?>
+<?php if (!extension_loaded("zlib")) die("skip zlib not available"); ?>
+--FILE--
+<?php
+$fname = str_replace('\\', '/', dirname(__FILE__) . '/files/HTML_CSS-1.5.4.tgz');
+try {
+	$tar = new PharData($fname);
+	$tar->extractTo(__DIR__ . '/files/ff');
```

Comment:
True, however extracting the files is not needed to prove the memory leak so I removed the extraction of files from the test.